system/index.template.scss

```diff
@@ -64,6 +65,7 @@ $scope: false;
   @include status;
   @include codeblock;
   @include party;
+  @include map;
```

Comment:
Check I've put the right `import`s and `include`s in the right places

Reply:
You mean check that we've done this properly in the other projects? We will need to add this to the SCSS file for each project.

Reply:
Nope. More basic/literal :smile: 

I've put some lines in `system/docs.scss` and `system/index.template.scss`, but I've not put anything in `system/system-core.scss`.

Have I done that right? (just in terms of this repo in isolation)

Reply:
Ah I see, yes, this looks fine